nodeJS/express-basics/Express-Introduction.md

```diff
@@ -35,6 +35,7 @@ By the end of this lesson, you should be able to do the following:
 
 
 ### Additional Resources
+This section contains helpful links to other content. It isn't required, so consider it supplemental for if you need to dive deeper into something.
```

Comment:
I browsed back and checked other pervious lessons. The line they used is "This section contains helpful links to other content. It isn’t required, so consider it supplemental for if you need to dive deeper into something."

If you could change it to that I think it'd be best!